Sources/EpoxyBars/BarModel/InternalBarModeling.swift

```diff
@@ -24,6 +24,9 @@ protocol InternalBarModeling: Diffable, EpoxyModeled {
 
   /// Should inform consumers that this bar model has been displayed.
   func didDisplay(_ view: UIView, traitCollection: UITraitCollection, animated: Bool)
+
+  /// Should inform consumers that this bar model has been selected.
+  func didSelect(_ view: UIView, traitCollection: UITraitCollection, animated: Bool)
```

Comment:
Ideally we'd also add a `isSelectable` is the same style as `InternalItemModeling` so that we only highlight if the model has an associated `didSelect` closure